supertokens_python/recipe/thirdparty/recipe.py

```diff
@@ -70,6 +81,20 @@ def __init__(self, recipe_id: str, app_info: AppInfo,
         self.api_implementation: APIInterface = api_implementation if self.config.override.apis is None else \
             self.config.override.apis(api_implementation)
 
+        email_delivery_ingredient = ingredients.email_delivery if ingredients is not None else None
+        self.email_delivery_ingredient = EmailDeliveryIngredient(self.config.get_email_delivery_config(recipe_implementation)) if email_delivery_ingredient is None else email_delivery_ingredient
+
+        if email_verification_recipe is not None:
+            self.email_verification_recipe = email_verification_recipe
+        else:
+            ev_email_delivery_ingredient = cast(
+                EmailDeliveryIngredient[TypeEmailVerificationEmailDeliveryInput],
+                self.email_delivery_ingredient
+            )
```

Comment:
Why cast?

Reply:
Yeah. It's not required here. Removed cast. Thanks!